lms/templates/problem.html

```diff
@@ -36,7 +36,10 @@ <h3 class="hd hd-2 problem-header" tabindex="-1">
     % endif
     % if answer_available:
     <span class="problem-action-button-wrapper">
-        <button class="show problem-action-btn btn-default btn-small"><span class="icon fa fa-exclamation-triangle" aria-hidden="true"></span><span class="show-label">${_('Show Answer')}</span></button>
+        <button class="show problem-action-btn btn-default btn-small">
+            <span class="icon fa fa-exclamation-triangle" aria-hidden="true"></span><span class="show-label">${_('Show Answer')}</span>
+            <span class="show-answer-tooltip sr">${_('To get credit, only select after Submit.')}</span>
```

Comment:
Given the concerns, I'd like to remove the tooltip from this PR and create a separate ticket for our backlog (so this can become unblocked, and because it is not an Oct requirement). We can then work with prod/ux/docs on best approach.

FYI @alisan617 , @chris-mike , @cahrens